recipes-apps/ovmenu-ng/files/openvario-57-lvds_DS2/ovmenu-ng.sh

```diff
@@ -0,0 +1,396 @@
+	
+	response=$?
+	case $response in
+		0) ;;
+		*) return 0
+	esac
+		
+	echo "Calibrating Sensors ..." >> /tmp/tail.$$
+	systemctl stop sensord
+	/opt/bin/sensorcal -c > /tmp/tail.$$
+
+	if [ $? -eq 2 ]
+	then
+		# board not initialised
+		dialog --backtitle "Openvario" \
+		--begin 3 4 \
+		--defaultno \
+		--title "Init Sensorboard" --yesno "Sensorboard is virgin ! \n Do you want to initialize ??" 10 40
+	
+		response=$?
+		case $response in
+			0) /opt/bin/sensorcal -i > /tmp/tail.$$
+			;;
+		esac
+		echo "Please run sensorcal again !!!" > /tmp/tail.$$
+	fi
+	sync
+	dialog --backtitle "OpenVario" --title "Result" --tailbox /tmp/tail.$$ 30 50
+	systemctl start sensord
+}
+
+function calibrate_touch() {
+	echo "Calibrating Touch ..." >> /tmp/tail.$$
+	/usr/bin/ov-calibrate-ts.sh >> /tmp/tail.$$
+	dialog --msgbox "Calibration OK!" 10 50
+}
+
+# Copy /usb/usbstick/openvario/maps to /home/root/.xcsoar
+# Copy only xcsoar-maps*.ipk and *.xcm files
+function update_maps() {
+	echo "Updating Maps ..." > /tmp/tail.$$
+	/usr/bin/update-maps.sh >> /tmp/tail.$$ 2>/dev/null &
+	dialog --backtitle "OpenVario" --title "Result" --tailbox /tmp/tail.$$ 30 50
+}
+
+# Copy /home/root/.xcsoar to /usb/usbstick/openvario/download/xcsoar
+function download_files() {
+	echo "Downloading files ..." > /tmp/tail.$$
+	/usr/bin/download-all.sh >> /tmp/tail.$$ &
+	dialog --backtitle "OpenVario" --title "Result" --tailbox /tmp/tail.$$ 30 50
+}
+
+# Copy /home/root/.xcsoar/logs to /usb/usbstick/openvario/igc
+# Copy only *.igc files
+function download_igc_files() {
+	/usr/bin/download-igc.sh
+}
+
+
+# Copy /usb/usbstick/openvario/upload to /home/root/.xcsoar
+function upload_files(){
+	echo "Uploading files ..." > /tmp/tail.$$
+	/usr/bin/upload-xcsoar.sh >> /tmp/tail.$$ &
+	dialog --backtitle "OpenVario" --title "Result" --tailbox /tmp/tail.$$ 30 50
+}
+
+function start_xcsoar() {
+	/usr/bin/xcsoar_config.sh
+	if [ -z $XCSOAR_LANG ]; then
+		/opt/XCSoar/bin/xcsoar -fly -640x480
+	else
+		LANG=$XCSOAR_LANG /opt/XCSoar/bin/xcsoar -fly -640x480
+	fi
+	sync
+}
+
+function yesno_exit(){
+	dialog --backtitle "Openvario" \
+	--begin 3 4 \
+	--defaultno \
+	--title "Really exit ?" --yesno "Really want to go to console ??" 5 40
+
+	response=$?
+	case $response in
+		0) echo "Bye";exit 1;;
+	esac
+}
+
+function yesno_restart(){
+	dialog --backtitle "Openvario" \
+	--begin 3 4 \
+	--defaultno \
+	--title "Really restart ?" --yesno "Really want to restart ??" 5 40
+
+	response=$?
+	case $response in
+		0) reboot;;
+	esac
+}
+
+function yesno_power_off(){
+	dialog --backtitle "Openvario" \
+	--begin 3 4 \
+	--defaultno \
+	--title "Really Power-OFF ?" --yesno "Really want to Power-OFF" 5 40
+
+	response=$?
+	case $response in
+		0) shutdown -h now;;
+	esac
+}
+
+setfont default8x16
+
+DIALOG_CANCEL=1 dialog --nook --nocancel --pause "Starting XCSoar ... \\n Press [ESC] for menu" 10 30 $TIMEOUT 2>&1
+
+case $? in
+	0) start_xcsoar;;
+	*) main_menu;;
+esac
+main_menu
```

Comment:
Did you change anything in this file or just copy-pasted it?

Reply:
I copy-paste these files for the new machines. They are necessary for compiling and I don't know an other method